code/drasil-example/Drasil/GamePhysics/Body.hs

```diff
@@ -58,8 +58,10 @@ import Data.Drasil.SentenceStructures (foldlSent, foldlSent_, foldlList,
   SepType(Comma), FoldType(List), sOf,sAnd, sOr, foldlSentCol, foldlSP, 
   foldlSPCol, showingCxnBw)
 import Data.Drasil.Software.Products (videoGame, openSource, sciCompS)
-import Data.Drasil.Utils (makeTMatrix, itemRefToSent, makeListRef, bulletFlat,
-  bulletNested, enumSimple, enumBullet)
+import Data.Drasil.Utils (makeTMatrix, itemRefToSent,
+  makeListRef, bulletFlat, bulletNested, enumSimple, enumBullet)
+import Language.Drasil.Printers (PrintingInformation(..),
+ Notation(Scientific, Engineering), defaultConfiguration)
```

Comment:
I think lines 63--64 should just be
```haskell
import Language.Drasil.Printers (PrintingInformation, defaultConfiguration)
```
[You should make sure that your PR is warning-free!]

Reply:
Warnings have been removed in d10ed89e52b0987b98192b916716fe095b1f85d1.